process/graduation_checklist.md

```diff
@@ -0,0 +1,31 @@
+# ASWF Project Graduation Template
+
+Text below in _italics_ are guidelines for answering the various questions.
+
+- **Project Name**:
+
+- **Criteria**
+
+  - **Demonstrate having a healthy number of committers from a diverse contributor base.**
+
+  _Showcase the total number of contributiors since the project entered incubation, as well as the diversity of contributors from an organizational standpoint. Pointing to GitHub or Dev Anayltics graphs are good ideas ( i.e. https://github.com/imageworks/OpenColorIO/graphs/contributors )_
```

Comment:
It would be helpful to clarify what each term means, i.e. contributors vs committers. As I understand it, a contributor is anyone making code changes while a committer is someone who can approve PRs and merge code, is that right?

If so -- should there be some data included here about committers? This seems to be more of a data point on the general community of contributors, which is fine IMO but this checklist item could be clarified on that point.

Reply:
Good point @bcipriano - if you feel comfortable take a crack at the right verbiage in here.

Reply:
Thinking about this more, it seems like a discussion point for a future TAC meeting. The criterion is a bit unclear here -- do we care only about contributors (as seems to be understood by most TAC/TSC members), do we care about committers, some mixture of the two?

I can take a stab at rephrasing the italic answer section here, but since the criterion text itself comes from an already-approved TAC doc it probably needs some brief discussion.

I missed this last TAC meeting due to travel but I'll make a note to add a discussion point for next time.